src/panels/RegisterController/RegisterController.tsx

```diff
@@ -181,10 +207,12 @@ export default function RegisterController() {
           <div className="col-10">
             <div className="p-form__control">
               <input
+                disabled={formValues.identityProvider}
```

Comment:
I was thinking that maybe adding `identityProvider: false` to the initial values of `formValues` might make this line more explicit?

Reply:
Done.